core/trino-main/src/main/java/io/trino/sql/planner/planprinter/JsonRenderer.java

```diff
@@ -51,19 +56,50 @@ private JsonRenderedNode renderJson(PlanRepresentation plan, NodeRepresentation
                 node.getIdentifier(),
                 node.getOutputs(),
                 node.getDetails(),
+                node.getReorderJoinStatsAndCost(),
+                getEstimates(plan, node),
                 children,
                 node.getRemoteSources().stream()
                         .map(PlanFragmentId::toString)
                         .collect(toImmutableList()));
     }
 
+    private List<PlanNodeStatsAndCostSummary> getEstimates(PlanRepresentation plan, NodeRepresentation node)
```

Comment:
Can we put this as a static method in `Renderer` and make `TextRenderer#printEstimates` use it ?